app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="well well-sm">
+  <div>
+  <h5><%= current_user.email %></h5>
+  </div>
+</div>
+
+<p>
+<div class="col-md-8">
+  <%= form_for [@user, @item] do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name, placeholder: "New List"  %>
+    <%= f.submit "Save" %>
+  <% end %>
+    <div class="form-group" >
+    </div>
+  </div>
+</p>
+
+<div class="container">
+  <div class="col-md-8">
+    <table>
+      <% @items.each do |i| %>
+        <tr>
+          <td><%= @item.name %></td>
+          <td><%= link_to @items_path %></td>
```

Comment:
Why don't we change it to `<%= link_to @item.name, [@item.user, @item] %>`?

Then you can remove the line above it for showing the name of the item.